asset-metadata/README.md

```diff
@@ -0,0 +1,54 @@
+## Web Asset Metadata
+ 
+Metadata is a special property on Web asset,
+that will enable extra headers sent from the device to the 
+target page of web asset.
+
+These extra headers mainly include screen related information,
+like hostname, screen name, location, etc.
```

Comment:
No need for etc. here. We have a full list of headers we expose. Should list all of them.